packages/website/content/pages/w3link.json

```diff
@@ -0,0 +1,359 @@
+{
+  "seo": {
+    "title": "",
+    "description": ""
+  },
+  "og": {
+    "site_name": "",
+    "url": "",
+    "type": "website",
+    "image": ""
+  },
+  "page_content": [
+    [
+      {
+        "type": "section-data",
+        "gradient": true,
+        "variant": "light"
+      },
+      {
+        "id": "product_hero-header",
+        "columns": [
+          {
+            "type": "custom",
+            "cols": {
+              "num": "col"
+            },
+            "customizations": [
+              {
+                "type": "image_block",
+                "id": "w3link-logo",
+                "src": "/images/logos/w3link-logo.svg"
+              }
+            ]
+          },
+          {
+            "type": "hero",
+            "page": "w3link",
+            "cols": {
+              "num": "col col-12",
+              "push_left": ""
+            },
+            "format": "header",
+            "heading": "Lightning fast reads from the IPFS network",
+            "subheading": "Read data available on the public IPFS network with <strong>w3link</strong> using an HTTP endpoint (at https://w3s.link), no additional software necessary. Add your content ID (CID) below to try it out."
+          },
+          {
+            "type": "w3link_gateway",
+            "cols": {
+              "num": "col col-12",
+              "push_left": ""
+            }
+          }
+        ]
+      }
+    ],
+    [
+      {
+        "id": "w3link_how",
+        "columns": [
+          {
+            "type": "text_block",
+            "cols": {
+              "num": "col"
+            },
+            "format": "small",
+            "heading": "The best of both worlds",
+            "description": [
+              "IPFS unlocks a ton of value at the data layer, but its peer-to-peer nature means reading data from the network isn’t always performant by today’s web standards. w3link marries IPFS content IDs with CDNs to provide a much more performant read experience - and is especially fast for content stored on the Web3.Storage platform - while still taking advantage of the benefits of IPFS.",
+              "In addition, like other IPFS HTTP gateways, w3link takes requests over HTTP, making it easy to integrate with today’s existing web infrastructure."
+            ],
+            "cta": {
+              "url": "https://docs.ipfs.tech/concepts/ipns/",
```

Comment:
Once https://github.com/web3-storage/web3.storage/pull/1736 is merged in:
docs/concepts/w3link/